exodus_lambda/functions/base.py

```diff
@@ -1,18 +1,32 @@
 import json
+import logging.config
+import logging
 
 
 class LambdaBase(object):
-    def __init__(self, conf_file="lambda_config.json"):
+    def __init__(self, logger_name="default", conf_file="lambda_config.json"):
         self._conf_file = conf_file
         self._conf = None
+        self._logger_name = logger_name
+        self._logger = None
 
     @property
     def conf(self):
         if not self._conf:
             with open(self._conf_file, "r") as json_file:
                 self._conf = json.load(json_file)
-
         return self._conf
 
+    @property
+    def logger(self):
+        if not self._logger:
+            logging.config.dictConfig(self.conf.get("logging") or {})
+            if self.conf["logging"]["loggers"][self._logger_name].get(
```

Comment:
What's the purpose of this branch?  Why do we want to disable all handlers on the root logger, isn't that going to remove our ability to collect log messages from any loggers other than the ones we listed in config?  Like warnings from one of the Amazon libraries?